static/app/components/modals/addDashboardWidgetModal.tsx

```diff
@@ -626,6 +627,7 @@ class AddDashboardWidgetModal extends React.Component<Props, State> {
             </StyledField>
           </DoubleFieldWrapper>
           {organization.features.includes('issues-in-dashboards') &&
+            source === DashboardWidgetSource.DASHBOARDS &&
```

Comment:
hmm should we also show this is `source === DashboardWidgetSource.LIBRARY`

Reply:
Yup for sure, good catch! Updated and added tests to `addDashboardWidgetModal.spec.jsx`